lib/workers/pr/index.ts

```diff
@@ -1,5 +1,7 @@
+import ignore from 'ignore';
```

Comment:
Why not using `minimatch`, as we already have it as a dependency?

Reply:
.ignore files don't like normal glob patterns. See also https://www.npmjs.com/package/ignore

> Pay attention that minimatch does not work in the gitignore way. To filter filenames according to .gitignore file, I recommend this module.